packages/adapter-static/index.js

```diff
@@ -50,7 +50,7 @@ export default function ({ pages = 'build', assets = pages, fallback, precompres
  * @param {string} directory
  */
 async function compress(directory) {
-	const files = await glob('**/*.{html,js,json,css,svg,xml}', {
+	const files = await glob('**/*.{html,js,json,css,svg,xml,wasm}', {
```

Comment:
It looks like this line also exists in `adapter-node`. We'd probably want to keep them in sync
https://github.com/sveltejs/kit/blob/e364454f6f7f057c758aab197e6015b6b93f287d/packages/adapter-node/index.js#L81